adoc/MAIN.release-notes.adoc

```diff
@@ -15,7 +15,71 @@ The most recent version of the Release Notes is available online at https://www.
 
 Entries can be listed multiple times if they are important and belong to multiple sections.
 
-Release notes usually only list changes that happened between two subsequent releases. Certain important entries from the release notes documents of previous product versions may be repeated. To make such entries easier to identify, they contain a note to that effect.
+Release notes usually only list changes that happened between two subsequent releases.
+Certain important entries from the release notes documents of previous product versions may be repeated.
+To make such entries easier to identify, they contain a note to that effect.
+
+== Changes in 4.1.0
+
+{productname} now ships with {kube} {kube_version}.
+Most of the significant changes relate to this upgrade, as more than 31 enhancements were merged in the {kube} {kube_version} release.
+You can read a short summary of the changes under [[kubernetes_updates]].
+Manual actions are required for 4.1.0 release.
+
+=== Required Actions
+
+==== skuba update instructions
+
+Update skuba on your management workstation as you would do with any other package.
+
+Refer to: link:https://documentation.suse.com/sles/15-SP1/single-html/SLES-admin/#sec-zypper-softup[{sls} 15 SP1 Admin Guide: Updating Software with Zypper]
+
+==== Upgrade your {kube} cluster:
+
+. Check if upgrades are available:
+  ``skuba cluster upgrade plan``
+. Check upgrades at the node level
+  ``skuba node upgrade plan <nodename>``
+. Upgrade all control plane node, one by one, by running
+  ``skuba node upgrade apply`` on each of them.
```

Comment:
Isn't it mandatory to upgrade master nodes first? I think so

Reply:
Yes. That's what was written as "control plane". To be consistent with other places I moved to a "master"/"worker" terminology in the last patchset.